src/client/content_cao.cpp

```diff
@@ -1593,6 +1593,7 @@ void GenericCAO::processMessage(const std::string &data)
 		bool sneak = !readU8(is);
 		bool sneak_glitch = !readU8(is);
 		bool new_move = !readU8(is);
+		bool new_jump = !readU8(is);
```

Comment:
the logic here and in `player_sao.cpp` needs to be inverted too, to properly make it false by default

Reply:
Inversion is now fixed